app/controllers/order_controller.rb

```diff
@@ -0,0 +1,26 @@
+class OrderController < ApplicationController
```

Comment:
@domenic99 as far as you added a controller with a right name, you can now destroy this.